packages/gutenframe/package.json

```diff
@@ -0,0 +1,33 @@
+{
+	"name": "@automattic/gutenframe",
+	"version": "1.0.0-alpha.0",
+	"description": "Utilities for the block editor iframe connections",
+	"main": "dist/server.js",
+	"sideEffects": false,
+	"repository": {
+		"type": "git",
+		"url": "git://github.com/Automattic/wp-calypso.git",
+		"directory": "apps/gutenframe"
+	},
+	"private": true,
```

Comment:
Are you planning to publish this package to NPM?

Reply:
Nope! Which is another reason why I ended up moving the package to the `apps` dir.